private_gpt/server/chunks/chunks_service.py

```diff
@@ -68,6 +70,9 @@ def __init__(
         self.embedding_component = embedding_component
         self.storage_context = StorageContext.from_defaults(
             vector_store=vector_store_component.vector_store,
+            graph_store=graph_store_component.graph_store
+            if graph_store_component and graph_store_component.graph_store
+            else None,
```

Comment:
you can simplify this with just `graph_store_component.graph_store`, the component can't be `None`. The dependency injector would fail before that.